src/pages/dashboard.vue

```diff
@@ -186,8 +186,24 @@ const pendingAddress = computed(() => {
   return false
 })
 
-const isChangeAddressDisabled = computed(() => business.currentBusiness.adminFreeze || pendingAddress.value)
-const isChangeDirectorDisabled = computed(() => business.currentBusiness.adminFreeze)
+const hasProgressCorrections = computed(() => {
+  if (filings && filings.value && filings.value.length > 0) {
+    const corrections = filings.value.filter((filing) => {
+      return filing.name === FilingTypes.CORRECTION &&
+        (filing.status === FilingStatusE.DRAFT ||
+        filing.status === FilingStatusE.PENDING ||
+        filing.status === FilingStatusE.PENDING_CORRECTION)
+    })
+    return corrections.length > 0
+  }
+  return false
+})
+
+const isChangeAddressDisabled = computed(() =>
+  business.currentBusiness.adminFreeze || pendingAddress.value || hasProgressCorrections.value
+)
+
+const isChangeDirectorDisabled = computed(() => business.currentBusiness.adminFreeze || hasProgressCorrections.value)
```

Comment:
I think this ticket is only for the red border in the todo section. The 'Change' button in the address section will be handled in #24010. The screenshot shows all the mismatches, but we broke the original QA bug into different ticket. 

Also, the logic for `isChangeAddressDisabled` and `isChangeDirectorDisabled` seems wrong. 

https://github.com/bcgov/business-filings-ui/blob/3a62b2b470bf16c181b879057823ff795e5faf68/src/views/Dashboard.vue#L131-L145  it uses: `:disabled="!isAllowed(AllowableActions.ADDRESS_CHANGE)"`. The function `isAllowed()` is already implemented in the business store.

Reply:
The ticket I grabbed mentioned the change buttons still so I added handling for the case when a correction filing exists. (It felt large for a 1) The logic in isChangeAddressDisabled and isChangeDirectorDisabled was already there I just added the hasProgressCorrections flag

Reply:
We will handle the other cases in #24010